### PR TITLE
Name acronyms correctly

### PR DESCRIPTION
**Pull request by ravage84**
To improve the readability/clarity about common acronyms, either name them correctly, e.g. all in caps or use quotes.

### CODE REVIEW

**Comment by dereuromark** [comment]
Those are not abbreviations, but acronyms :)
But your point is still valid.

File: src/Controller/Component/RequestHandlerComponent.php [hunk 1]
```diff
@@ -380,13 +380,13 @@ public function prefers($type = null)
      *
      * ### Usage:
      *
-     * Render the response as an 'ajax' response.
+     * Render the response as an 'AJAX' response.
```
Comment (garas) [inline src/Controller/Component/RequestHandlerComponent.php:383]: Unquote or leave as it was before.

File: src/View/XmlView.php [hunk 2]
```diff
@@ -61,7 +61,7 @@
 class XmlView extends SerializedView
 {
     /**
-     * XML layouts are located in the xml sub directory of `Layouts/`
+     * XML layouts are located in the 'xml' sub directory of `Layouts/`
```
Comment (garas) [inline src/View/XmlView.php:64]: Use backticks `` `xml` `` as in `Layouts` or add slash `xml/`

Comment (garas) [reply src/View/XmlView.php:64]: And `Layouts/` are `layout/` in 4.x.

Comment (othercorey) [reply src/View/XmlView.php:64]: Just use `layouts/xml/`

**Pull request merged**